src/plugins/tables/tables.js

```diff
@@ -135,31 +135,40 @@ $document.on( "init.dt draw.dt", selector, function( event, settings ) {
 		ol = document.createElement( "OL" ),
 		li = document.createElement( "LI" );
 
-	// Update Pagination List
-	for ( var i = 0; i < paginate_buttons.length; i++ ) {
-		var item = li.cloneNode( true );
-		item.appendChild( paginate_buttons[ i ] );
-		ol.appendChild( item );
-	}
+	// Determine if Pagination required
+	if ( paginate_buttons.length < 4 ) {
+		$( ".dataTables_paginate" ).addClass( "hidden" );
```

Comment:
This line will remove pagination for all the datatables on the page. 

The following would fix the issue:

``` pagination.addClass( "hidden" ); ```